### PR TITLE
Add regression test for summary + completion variables

### PR DESCRIPTION
**Pull request by joakim-hove**
Enable regression test with `WORPL` and family keywords

### CODE REVIEW

**Comment by joakim-hove** [comment]
jenkins build this update_data please

**Comment by joakim-hove** [comment]
jenkins build this opm-tests=429 please

File: tests/update_reference_data.sh [hunk 1]
```diff
@@ -136,6 +136,7 @@ tests[pinch_multz_all]="flow pinch PINCH_MULTZ_ALL"
 tests[pinch_multz_all_barrier]="flow pinch PINCH_MULTZ_ALL_BARRIER"
 tests[model6_msw]="flow model6 1_MSW_MODEL6"
 tests[norne_reperf]="flow norne NORNE_ATW2013_B1H_RE-PERF"
+tests[compl_smry]="flow cmpl_smry COMPL_SMRY"
```
Comment (akva2) [inline tests/update_reference_data.sh:139]: cmpl_smry vs compl_smry

Comment (joakim-hove) [reply tests/update_reference_data.sh]: Thanks - fixed

**Comment by joakim-hove** [comment]
jenkins build this update_data please

File: tests/update_reference_data.sh (hunk 1 above)
Comment (akva2) [reply tests/update_reference_data.sh]: fairly certain you pushed things here you did not intend however..

Comment (joakim-hove) [reply tests/update_reference_data.sh]: ehhh - what a mess; thank you ;-)

Extra piggy-back commit removed.

**Comment by joakim-hove** [comment]
jenkins build this opm-tests=429 please

**Comment by joakim-hove** [comment]
jenkins build this update_data please

**Comment by akva2** [comment]
jenkins build this opm-tests=430 please

**Comment by joakim-hove** [comment]
> jenkins build this opm-tests=430 please

Thanks

**Pull request merged**